### PR TITLE
Dockerisation of app and integrations

### PR DESCRIPTION
**Pull request by naufalmirk**
https://companieshouse.atlassian.net/browse/BI-3645

This includes dockerising:
- Dissolution frontend app:
Integrations with docker-compose:
- CDN
- Accounts

Also updates to README for getting up and running locally.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 dist
 node_modules
 .idea
+src/**/*.js
```
Comment (peadarkelly) [inline .gitignore:4]: Why would there be any JS files in these directories?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    environment:
+      - ACCOUNT_CH_CLIENT_ID=${OAUTH2_CLIENT_ID}
+      - ACCOUNT_CH_CLIENT_SECRET=${OAUTH2_CLIENT_SECRET}
+      - ACCOUNT_OAUTH2_REQUEST_KEY=${OAUTH2_REQUEST_KEY}
+      - ACCOUNT_OAUTH2_TOKEN_KEY=${OAUTH2_TOKEN_KEY}
+      - ACCOUNT_URL=http://account.chs-dev
+      - API_LOCAL_URL=http://localhost?API_LOCAL_URL
+      - CACHE_SERVER=redis:6379
+      - CDN_HOST=cdn.chs-dev
+      - CHS_API_KEY
```
Comment (peadarkelly) [inline docker-compose.yml:77]: should this have some value? Or point to local bash profile using:
```
- CHS_API_KEY=${CHS_API_KEY}
```

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  cdn-ch-gov-uk:
+    container_name: chs-cdn-ch-gov-uk
+    image: 169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/cdn.ch.gov.uk:latest
+    labels:
+      - traefik.http.routers.cdn-ch-gov-uk.rule=Host(`cdn.chs-dev`)
+    networks:
+      - chs-dev
+    expose:
+      - 80
+  account-ch-gov-uk:
```
Comment (peadarkelly) [inline docker-compose.yml:64]: I think a redis container may also be needed to facilitate accounts. Damian has set some up on his POC. To test this, you should be able to access account.chs-dev/signin through the browser

File: .gitignore (hunk 1 above)
Comment (naufalmirk) [reply .gitignore:4]: not sure, I think the .gitignore came from appeals repo.

File: docker-compose.yml (hunk 3 above)
Comment (naufalmirk) [reply docker-compose.yml:64]: Ah ok, I'll integrate the redis part as well.

File: .gitignore (hunk 1 above)
Comment (mateuszc-kainos) [reply .gitignore:4]: I added that, I'll remove it.

**Pull request merged**